### DISCUSSION TITLE
Copy to clipboard as SVG and / or PNG?

### DISCUSSION THREAD

**Discussion by lobau** [uxbox/uxbox#1415] [Ideas]
It would be awesome if I could select a shape, and copy its path as SVG in my clipboard directly, so I can paste it somewhere directly without having to download a file, open it, and copy the code first.
Being able to copy a frame as PNG would be super helpful too, to paste it directly in Telegram or Slack for example.

**Comment by KronosTheLate** [comment]
Just to bring some life to this 3.5 year old issue, a big +1 from me. I would like to use penpot for sketching more than app layout development, and being able to share the sketches easily would be superuseful. If nothing else, making it so that right-clicking a board in the preview showed the option "copy image to clipboard" would be very useful.

**Comment by dodalovicgran** [comment]
+1
